testnet-contracts/contracts/BankToken.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.5.0;
+
+import "../../node_modules/openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol";
+
+/**
+ * @title BankToken
```

Comment:
Thinking about naming, this is basically a generic wrapper around ERC20Detailed and ERC20Mintable, so maybe just call it ERC20DetailedMintable?

Reply:
Renamed to CosmosToken in https://github.com/cosmos/peggy/pull/76/commits/89cb10e8d06ff3dcd467bcfc7b2821bf14240153. I think Bridge token is better as it is more generic and will update the naming conventions again.